StripeFinancialConnections/StripeFinancialConnections/Source/API Bindings/Models/FinancialConnectionsOAuthPrepane.swift

```diff
@@ -0,0 +1,73 @@
+//
+//  FinancialConnectionsOAuthPrepane.swift
+//  StripeFinancialConnections
+//
+//  Created by Krisjanis Gaidis on 1/6/23.
+//
+
+import Foundation
+@_spi(STP) import StripeUICore
+
+struct FinancialConnectionsOAuthPrepane: Decodable {
+
+    let institutionIcon: FinancialConnectionsImage?
+    let title: String
+    let body: OauthPrepaneBody
+    let partnerNotice: OauthPrepanePartnerNotice?
+    let cta: OauthPrepaneCTA
+    let dataAccessNotice: FinancialConnectionsDataAccessNotice
+
+    struct OauthPrepaneBody: Decodable {
+        let entries: [OauthPrepaneBodyEntry]?
+
+        struct OauthPrepaneBodyEntry: Decodable {
+
+            enum EntryType: String, SafeEnumCodable, Equatable {
+                case text
+                case image
+                case unparsable
+            }
+
+            let type: EntryType
+            private let content: Any?
```

Comment:
backend link for double-check:
- https://livegrep.corp.stripe.com/view/stripe-internal/pay-server/lib/bank_connections/text/data/oauth_prepane_body_entry.rb#L5-17

Reply:
Looks like a better way to represent this would be an enum with associated types. Similar to https://github.com/stripe/stripe-ios/blob/8e6363c7/StripeFinancialConnections/StripeFinancialConnections/Source/API%20Bindings/Models/FinancialConnectionsSession.swift#L61

Reply:
@vardges-stripe that is a novel way to structure it, thank you! I hadn't thought about that. It _feels_ a little weird as this is not how I typically view an enum, but it does work out nicely & the alternatives are not necessarily better